.gitignore

```
@@ -0,0 +1,3 @@
+node_modules
+testData
+lab
```

```suggestion
lab
.vs*
```
This line will ignore the .VSCode file